examples/unary-operators/unary_operators.bal

```diff
@@ -0,0 +1,24 @@
+import ballerina/io;
+
+public function main() {
+    int a = 10;
+
+    // Negate the value of `a`.
+    io:println(-a);
```

Comment:
Extract to variables to show static type. Preferably include an example where the type is not the same as the operand type. E.g., int subtype/byte